src/main/connector/hardware/MIB2-header/MIB2-header.yaml

```diff
@@ -11,6 +11,11 @@ connector:
     # Criteria(1): there must be something in the ifTable SNMP Table
     - type: snmpGetNext
       oid: 1.3.6.1.2.1.2.2.1
+metrics:
+  hw.network.dropped:

```

Comment:
This metric definition should go to semconv/Hardware.yaml and also defined in https://opentelemetry.io/docs/specs/semconv/system/hardware-metrics/#hwnetwork---network-adapter-metrics